test/e2e/serial/tests/workload_unschedulable.go

```diff
@@ -229,4 +229,118 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload unsched
 			}
 		})
 	})
+
+	Context("with at least two nodes with two numa zones and enough resources in one numazone", func() {
+		var requiredRes corev1.ResourceList
+		var targetNodeName string
+		BeforeEach(func() {
```

Comment:
for this specific case I'd just inline the `BeforeEach` in the `It` block. Or, if we want to keep it, we would need a bit stronger rationale